### PR TITLE
Use consistent classes-related variables

### LINKED ISSUE #354: Consistent variable names for classes
In general, we use `classes` for variables set in a template and `modifier_classes` for variables that can be modified when including/embedding templates. We should make sure all cases follow this convention. (e.g., base images)

### PR DESCRIPTION
**Pull request by dcmouyard**
Fixes #354. This PR updates all classes-related twig variable names to be consistent and adds modifier_classes to most components"

### CODE REVIEW

File: source/_patterns/04-components/skiplinks/skiplinks.twig [hunk 1]
```diff
@@ -1,3 +1,8 @@
+{% set classes = [
+  'skiplinks',
+  modifier_classes ? modifier_classes,
+]|join(' ')|trim %}
+
 <div class="skiplinks">
```
Comment (coreylafferty) [inline source/_patterns/04-components/skiplinks/skiplinks.twig]: Need to output the classes var here

File: source/_patterns/04-components/site-slogan/site-slogan.twig [hunk 2]
```diff
@@ -1 +1,6 @@
+{% set classes = [
+  'site-slogan',
+  modifier_classes ? modifier_classes,
+]|join(' ')|trim %}
+
 <span class="site-slogan">{{ site_slogan }}</span>
```
Comment (coreylafferty) [inline source/_patterns/04-components/site-slogan/site-slogan.twig]: need to output the classes var here

File: source/_patterns/04-components/site-name/site-name.twig [hunk 3]
```diff
@@ -1,3 +1,8 @@
+{% set classes = [
+  'site-name',
+  modifier_classes ? modifier_classes,
+]|join(' ')|trim %}
+
 <a class="site-name" href="{{ url }}" title="{{ 'Home'|t }}" rel="home">
```
Comment (coreylafferty) [inline source/_patterns/04-components/site-name/site-name.twig]: need to output the classes var here

**Review by coreylafferty — CHANGES_REQUESTED** [review]

**Comment by dcmouyard** [comment]
@coreylafferty The missing classes variables in those components should be fixed now.

**Review by coreylafferty — APPROVED** [review]

**Pull request merged**